### PR TITLE
fix formatting on api docs

### PR DESCRIPTION
**Pull request by mikehelmick**
## Proposed Changes

* previous formatting didn't read great on pkg.go.dev

**Release Note**

```release-note
NONE
```

### CODE REVIEW

File: pkg/api/v1/exposure_types.go [hunk 1]
```diff
@@ -68,9 +68,7 @@ const (
 
 // Publish represents the body of the PublishInfectedIds API call.
 // temporaryExposureKeys: Required and must have length >= 1 and <= 21 (`maxKeysPerPublish`)
-// healthAuthorityID: The identifier for the mobile application.
-//  - Android: The App Package AppPackageName
```
Comment (whaught) [inline pkg/api/v1/exposure_types.go:72]: Did you intend to remove `Android` and `IOS` or should we put those somewhere?

Comment (whaught) [reply pkg/api/v1/exposure_types.go:72]: /lgtm /hold

Comment (whaught) [reply pkg/api/v1/exposure_types.go:72]: /lgtm /hold

Comment (mikehelmick) [reply pkg/api/v1/exposure_types.go:72]: yes, that was leftover from v1alpha1

**Pull request merged**